ui/src/components/home/AfterLogin.tsx

```diff
@@ -0,0 +1,87 @@
+import User from '@/access/User';
+import Role from '@/access/Role';
+import Image from 'next/image';
+import React from 'react';
+import {KeyRound} from 'lucide-react';
+import UserInfoItem from '@/components/home/UserInfoItem';
+
+const AfterLogin = (
+    {
+        currentUser,
+        numberOfGroupings,
+        numberOfMemberships
+    }: {
+        currentUser: User,
+        numberOfGroupings: number,
+        numberOfMemberships: number
+    }) => {
+    const isAdmin = currentUser.roles.includes(Role.ADMIN);
+    const isOwner = currentUser.roles.includes(Role.OWNER);
+    const getHighestRole = () => isAdmin ? 'Admin' : isOwner ? 'Owner' : 'Member';
```

Comment:
Nested ternary operators can be confusing, instead use if, else if, and else. Thanks!

Reply:
done